howl/data/tokenize.py

```diff
@@ -101,14 +102,24 @@ def decode(self, ids: List[int]) -> str:
 
     def encode(self, transcript: str) -> List[int]:
         encoded_output = []
+        # append a space at the beginning to catch vocab starting with a space
+        # append a space to add the last negative label
+        transcript = " " + transcript + " "
```

Comment:
Do `f' {transcript} '` instead.

Reply:
And does the codebase use single quotes or double quotes? I would keep the style consistent.